bindings/python/tests/test_offline.py

```diff
@@ -97,24 +99,6 @@ def test_hex_utf8():
     assert hex_to_utf8(hex_data) == utf8_data
 
 
-def test_block():
```

Comment:
Why remove it and no skip it like the others ?

Reply:
because I basically moved it. IIRC this test computes the block id which is now done for the `BlockWrapper`. I simply added another assert to the already existing test to minimize some redundancy.

Reply:
Ah alright, I must have missed the move.